tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -913,6 +914,14 @@ def testTrtGraphConverter_AllowEngineNativeSegmentExecution(self):
     if not is_tensorrt_enabled():
       return
 
+    # This test will not work anymore with TRT >= 8. TensorRT does not
+    # preallocate anymore the max_workspace_size_bytes, but rather allocates as
+    # it needs up to this value. We need to fix this bug with TRT >= 8.
+    # NVBug: 3329724
```

Comment:
This NVBug is about testBackwardCompatibility. Why it is relevant here?

I believe the bug is not relevant. The comment here only needs to explain why the test doesn't work for TRT >=8.

Did I miss something?

Reply:
Mention to the NVBug removed in the comment as discussed